cockroachdb/Chart.yaml

```diff
@@ -2,8 +2,8 @@
 apiVersion: v1
 name: cockroachdb
 home: https://www.cockroachlabs.com
-version: 7.0.3
-appVersion: 21.2.10
+version: 7.0.4
```

Comment:
There is a bug in our script:

Can you bump `version` to `8.0.0`?

(i.e. a major version bump for `appVersion` should also mean a major version bump for `version`)

Thank you!

Reply:
Done!